js/src/secureApi.js

```diff
@@ -56,8 +62,8 @@ export default class SecureApi extends Api {
   }
 
   get hostname () {
-    if (window.location.hostname === 'home.parity') {
-      return 'dapps.parity';
+    if (window.location.hostname === 'home.ethlink.io') {
```

Comment:
Not 100% sure what this is doing here, however...

1. This is dependent on the ethlink branch (should be merged first)
2. We have moved to `web3.site`
3. We still have the proxy under settings, so this breaks that